### DISCUSSION TITLE
Grid Listing show Error

### DISCUSSION THREAD

**Discussion by ravi-vegda** [pimcore/pimcore#9803] [Q&A / Support]
Pimcore v10.0.7

I make one custom import command which will import approx 5000 records from CSV to pimcore.
import process was show success result. but when I back to pimcore and open any listing/Grid. pimcore show below error:

Timestamp: Mon Jul 19 2021 19:49:54 GMT+0530 (India Standard Time)
Status: 0 | communication failure
URL: /admin/object/grid-proxy?classId=PRODUCTS&folderId=1474&xaction=read&_dc=1626704364769
Method: GET
Message: 
undefined

![image](https://user-images.githubusercontent.com/86904617/126174928-cbeea128-b605-43c7-a008-304729171d07.png)

**Comment by Chauhan-Mukesh** [comment]
Hi, 
I am getting same error. Do you have any solution, any suggestions?
